docs/en/flight_controller/svehicle_e2.md

```diff
@@ -0,0 +1,167 @@
+# S-Vehicle E2
+
+:::warning
+PX4 does not manufacture this (or any) autopilot.
+:::
+
+The _E2_ is an advanced autopilot manufactured by S-Vehicle<sup>&reg;</sup>.
+
+The autopilot is recommended for commercial system integration, but is also suitable for academic research and any other applications.
+It brings you ultimate performance, stability, and reliability in every aspect.
+
+![SVehicle-E2](../../assets/flight_controller/svehicle_e2/main.png)
+
+::: info
+These flight controllers are [manufacturer supported](../flight_controller/autopilot_manufacturer_supported.md).
+:::
+
+### Processors & Sensors
+
+- FMU Processor: STM32H753IIK6
+  - 32 Bit Arm® Cortex®-M7, 480MHz, 2MB flash memory, 1MB RAM
+- IO Processor: STM32F103
+  - 32 Bit Arm® Cortex®-M3, 72MHz, 20KB SRAM
+- On-board sensors
+  - Accel/Gyro: BMI088
+  - Accel/Gyro: ICM-42688-P
+  - Accel/Gyro: ICM-20649
+  - Mag: RM3100
+  - Barometer: 2x ICP-20100
+
+### Interfaces
+
+- 14x PWM Servo Outputs
+- 1x Dedicated R/C Input for Spektrum / DSM and S.Bus
+- 1x Analog/PWM RSSI Input
+- 2x TELEM Ports (with full flow control)
+- 1x UART4 Port
+- 2x GPS Ports
+  - 1x Full GPS plus Safety Switch Port (GPS1)
+  - 1x Basic GPS Port (with I2C, GPS2)
+- 1x USB Port (TYPE-C)
+- 1x Ethernet Port
+  - Transformerless application
+  - 100Mbps
+- 3x I2C Bus Ports
+- 1x SPI Bus
+  - 1x Chip Select Line
+  - 1x Data Ready Line
+  - 1x SPI Reset Line
+- 2x CAN Ports
+- 3x Power Input Ports
+  - ADC Power Input
+  - I2C Power Input
+  - DroneCAN/UAVCAN Power Input
+- 2x AD Ports
+  - Analog Input (3.3V)
+  - Analog Input (6.6V - not supported by PX4)
+- 1x Dedicated Debug Port
+  - FMU Debug
+
+## Purchase Channels
+
+Order from [S-Vehicle](https://svehicle.cn/).
+
+## Serial Port Mapping
+
+| UART   | Device     | Port          |
+| ------ | ---------- | ------------- |
+| USART1 | /dev/ttyS0 | GPS           |
+| USART2 | /dev/ttyS1 | TELEM3        |
+| USART3 | /dev/ttyS2 | Debug Console |
+| UART4  | /dev/ttyS3 | UART4         |
+| UART5  | /dev/ttyS4 | TELEM2        |
+| USART6 | /dev/ttyS5 | PX4IO/RC      |
+| UART7  | /dev/ttyS6 | TELEM1        |
+| UART8  | /dev/ttyS7 | GPS2          |
+
+## PWM Output
+
+The E2-Plus flight controller supports up to 14 PWM outputs.
+The first 8 outputs (labelled M1 to M8) are controlled by a dedicated STM32F103 IOMCU controller.
+The remaining 6 outputs (labelled 9 to 16) are the "auxiliary" outputs.
+These are directly attached to the STM32H753 FMU controller .
+
+The 14 PWM outputs are:
+
+M1 - M8 are connected to the IOMCU
+A1 - A6 are connected to the FMU
+
+M1 - M8 support DShot and are in 3 groups:
```

Comment:
No they don't on PX4. On PX4 Dshot must be connected to the FMU https://docs.px4.io/main/en/peripherals/dshot.html#wiring